.github/workflows/release.yaml

```diff
@@ -58,6 +58,14 @@ jobs:
         cargo -V
         rustc -V
 
```

Comment:
I'm happy with the rest, but can we remove this one?

This workflow is only ever triggered when I push a release tag, and for *paranoid reasons* I'd like to ensure releases are built from scratch rather than having dependencies loaded from the cache.